oak_functions_service/tests/integration_test.rs

```diff
@@ -20,9 +20,6 @@
 extern crate alloc;
 
 use core::assert_matches::assert_matches;
-use std::sync::Arc;
-
```

Comment:
It looks like this uses an older version of formatting. Since #4648 it rustfmt should automatically group use statements.

Reply:
rustfmt is actively adding a space there. I think it's adding a space dividing external imports from workspace imports.